docs/getting-started/README.md

```diff
@@ -35,14 +35,30 @@ docker compose build --no-cache client
 ## Start the client
 
 ```bash
-docker compose up client
+docker compose up [-d] client
 ```
 
-After initialization, the client is running at `http://localhost:${DJANGO_LOCAL_PORT}`
-(<http://localhost:8000> by default).
+The optional `-d` flag will start in _detatched_ mode and allow you to continue using the terminal session. Otherwise your
+terminal will be attached to the container's terminal, showing the startup and runtime output.
 
-If `DJANGO_ADMIN=true`, the backend administrative interface can be accessed with the superuser you setup at
-<http://localhost:8000/admin>.
+After initialization, the client is running running on `http://localhost` at a port dynamically assigned by Docker. To find the
```

Comment:
What the Docker Desktop interface looks like: Click **Containers / Apps**, 
![image](https://user-images.githubusercontent.com/3673236/135665627-c84ac98e-dfba-4856-8981-13874d85b41e.png)

![image](https://user-images.githubusercontent.com/3673236/135665545-1d243d13-aec3-4cfe-9db0-f7e8cc6fdec5.png)

Find $PORT at `http://0.0.0.0:56760/` or `http://localhost:56760/`